script/02_DeployCLPoolManager.s.sol

```diff
@@ -5,24 +5,49 @@ import "forge-std/Script.sol";
 import {BaseScript} from "./BaseScript.sol";
 import {IVault} from "../src/interfaces/IVault.sol";
 import {CLPoolManager} from "../src/pool-cl/CLPoolManager.sol";
+import {Create3Factory} from "pancake-create3-factory/src/Create3Factory.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
 /**
+ * Step1:
  * forge script script/02_DeployCLPoolManager.s.sol:DeployCLPoolManagerScript -vvv \
  *     --rpc-url $RPC_URL \
  *     --broadcast \
- *     --slow \
- *     --verify
+ *     --slow
+ *
+ * Step2:
+ * # Get the ABI-encoded form of the constructor arguments
+ * cast abi-encode "CLPoolManager(address)" <vault_addr>
+ *
+ * Step3:
+ * forge verify-contract <address> CLPoolManager --watch --chain <chain_id> \
```

Comment:
double check, do we really need to `abi-encode`? 

```
// or we can do this?
--constructor-args 0x9E49a3c00B458d740F6370f90B781af1dbCdF8C5
```

Reply:
no, we can't do this. Can see the usage  

![CleanShot 2024-10-30 at 09 45 31@2x](https://github.com/user-attachments/assets/e4d0a6dd-e8a4-4c87-b8f4-9b298b1d8d37)